### ISSUE TITLE
Decide on a license

### ISSUE THREAD

**Issue by rkujawa** [Sakura-IT/SonnetAmiga#2]
The source code of this project is provided here on GitHub, but it is not licensed. We have to decide under what license will the program be distributed.

There are two common options in the open source world:
- BSD license (typically 2-clause version is used for new projects): http://opensource.org/licenses/BSD-2-Clause
- GNU General Public License (newest version is 3): http://www.gnu.org/copyleft/gpl.html

We should carefully think about which license will be better, knowing the peculiarities of Amiga community.

**Comment by DvdBoon** [comment]
Contacted Sam Jordan. We can re-use some code. Still have to contact H&P though, who are the actual copyright holders. If it is denied then parts of ppcfunctions.p need to be rewritten before a license can be agreed upon.

**Comment by rkujawa** [comment]
Do we still have offending code in ppcfuncions.p?

**Comment by DvdBoon** [comment]
Yes. But on the other hand, some are already direct copies (but then in PPC assembly) from exec functions.

**Comment by rkujawa** [comment]
Can you pinpoint which of these functions really need rewriting?

**Comment by rkujawa** [comment]
Also, a temporary solution would be to just apply the license at the top of files that are "clean" from copyright point of view.

**Comment by DvdBoon** [comment]
That's everything except ppcfunctions.p

Some things have been the result of looking how copyrighted code behaved,
 like pci.i and pci_lib.i and the fd.

I'm guessing Sonnet_lib.fd is also partly based on the powerpc library one..

Op woensdag 10 mei 2017 heeft Radosław Kujawa <notifications@github.com>
het volgende geschreven:

**Comment by rkujawa** [comment]
Things that are a result of reverse engineering ("looking how copyrighted code behaved") do not pose any copyright problem.

In the current situation I suggest going with MIT License for the purpose of this project (essentially, an equivalent of 2-clause BSD license).
https://en.wikipedia.org/wiki/MIT_License

If you are OK with this, I will slap it on top of every source file, with the exception of ppcfunctions.p and wosdb stuff.

**Comment by DvdBoon** [comment]
ok

**Comment by rkujawa** [comment]
Slapped MIT license on top of source files (d3bac3a).
